test/extended/cmd.sh

```diff
@@ -54,88 +54,89 @@ oadm policy add-role-to-group view system:authenticated -n default
 
 install_registry
 wait_for_registry
-DOCKER_REGISTRY=`oc get service/docker-registry -n default -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}'`
+docker_registry="$( oc get service/docker-registry -n default -o jsonpath='{.spec.clusterIP}:{.spec.ports[0].port}' )"
```

Comment:
It's one of the style things - `$ALL_CAPS` are globals from other files. This greatly aids in debugging when you are trying to find the source of a variable.